src/app/listener.ts

```diff
@@ -42,6 +45,29 @@ export class Listener {
 
       this.container.loggerService.info(`Loaded ${this.container.jobService.size()} jobs...`);
 
+      const commands = Object.entries(this.container.pluginService.plugins)
+        .filter((entry) => {
+          const [, plugin] = entry;
+          return isSlashCommand(plugin);
+        })
+        .map((entry) => {
+          const [name, plugin] = entry;
+          const options = isSlashCommand(plugin)
+            ? (plugin as unknown as ISlashPlugin).parameters
+            : undefined;
```

Comment:
use null instead for user-specified "undef"

Reply:
This breaks the typings on line 68